### PR TITLE
Updates to FamilyMemberHistory

### PR DESCRIPTION
**Pull request by sydneanschutz**
Description
----
Updates to FamilyMemberHistory

Screen Shot 2021-09-17 at 10 02 47 AM
Screen Shot 2021-09-17 at 10 02 51 AM
Screen Shot 2021-09-17 at 10 03 03 AM
Screen Shot 2021-09-17 at 10 03 22 AM
Screen Shot 2021-09-17 at 10 03 48 AM

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 1]
```diff
@@ -231,6 +235,11 @@ Update a FamilyMemberHistory.
 
     PUT /FamilyMemberHistory/:id
 
+_Implementation Notes_
+
+* Conditions can be removed from a FamilyMemberHistory by setting the 'condition-lifecycle-status' modifierExtension to 'entered-in-error'. Only existing conditions can be removed; condition id must be provided.
+* An update should only be preformed directly after a FamilyMemberHistory is retrieved. If a condition.id is provided from the get operation, it MUST be provided on the subsequent update and must match the condition.id returned from the get.
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/summary/family-member-history.md:241]: should be mention some how some where that a condition.code must be unique? like u cant add the same condition twice? 

And something about certain relationships only being provided once 

Like they will get unhelpful 400 if they do either of those things

File: content/millennium/r4/clinical/summary/family-member-history.md [hunk 2]
```diff
@@ -231,6 +235,11 @@ Update a FamilyMemberHistory.
...
+
 ### Authorization Types
 
 <%= authorization_types(provider: true, system: true, patient: false) %>
```
Comment (sydneanschutz) [inline content/millennium/r4/clinical/summary/family-member-history.md:245]: Should we mention that if you're trying to change the status of a FMH to health-unknown or completed with no conditions.... that u must ensure all conditions are removed first 
I think im thinking about that right? they will again get a generic 400 if they dont remove the conditions first

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (MadhurRaj) [reply content/millennium/r4/clinical/summary/family-member-history.md:241]: Let's add that each item in the list must represent a distinct condition.

I don't think we can provide a full list of which relationships should only be provided once, but I think it's fairly straight forward. So let's leave that one off.

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 2 above)
Comment (MadhurRaj) [reply content/millennium/r4/clinical/summary/family-member-history.md:245]: Hmm...let's leave it off for now. I think we'll need to come up with a good way of explaining the workflows with the specific values, which we can add a little later.

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md:241]: Screen Shot 2021-09-22 at 3 36 02 PM
I put it here, is that a good place?
https://github.com/cerner/fhir.cerner.com/pull/732/commits/0ec200035c4073e7f7f61541d5d2eb5545e56893#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63R290

File: content/millennium/r4/clinical.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ layout: api
 * [AllergyIntolerance](/millennium/r4/clinical/summary/allergy-intolerance)
 * [Condition](/millennium/r4/clinical/summary/condition)
 * [Procedure](/millennium/r4/clinical/summary/procedure)
+* [FamilyMemberHistory](/millennium/r4/clinical/summary/family-member-hisory)
```
Comment (RayGunY) [inline content/millennium/r4/clinical.md:14]: 'history'

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (RayGunY) [inline content/millennium/r4/clinical/summary/family-member-history.md:241]: 'performed'

Comment (MadhurRaj) [reply content/millennium/r4/clinical/summary/family-member-history.md:241]: yea that's fine

File: lib/resources/r4/family_member_history.yaml [hunk 4]
```diff
@@ -203,7 +203,8 @@ fields:
     - update
   note: |
     <ul>
-      <li>When deceased field is not provided, it is defaulted to deceasedBoolean false. </li>
+      <li>When deceased field is not provided, it is defaulted to deceasedBoolean false.</li>
+      <li>When deceasedAge is provodied without the precision extension, it is defaulted to 'Age'</li>
```
Comment (RayGunY) [inline lib/resources/r4/family_member_history.yaml:207]: 'provided'

File: content/millennium/r4/clinical.md (hunk 3 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical.md]: https://github.com/cerner/fhir.cerner.com/pull/732/commits/2b9dbafaf8d103502e3951c22745a5718eab499c#diff-62ccb408b63ccfbcb9845290842d4fb0439e12939fd104933c9af7b2ed98a96dL14

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/732/commits/2b9dbafaf8d103502e3951c22745a5718eab499c#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68L241

File: lib/resources/r4/family_member_history.yaml (hunk 4 above)
Comment (sydneanschutz) [reply lib/resources/r4/family_member_history.yaml]: https://github.com/cerner/fhir.cerner.com/pull/732/commits/2b9dbafaf8d103502e3951c22745a5718eab499c#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L207

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (MadhurRaj) [inline content/millennium/r4/clinical/summary/family-member-history.md]: for the get, let's say

condition.id is returned on the get operation.

File: lib/resources/r4/family_member_history.yaml [hunk 5]
```diff
@@ -330,6 +336,10 @@ fields:
   description: The age of onset.
   url: https://hl7.org/fhir/familymemberhistory-definitions.html#FamilyMemberHistory.condition.onset_x_
   action: update
+  note: |
+    <ul>
+      <li>When onsetAge is provodied without the precision extension, it is defaulted to 'Age'</li>
```
Comment (RayGunY) [inline lib/resources/r4/family_member_history.yaml]: 'provided'

File: content/millennium/r4/clinical/summary/family-member-history.md (hunk 1 above)
Comment (sydneanschutz) [reply content/millennium/r4/clinical/summary/family-member-history.md]: https://github.com/cerner/fhir.cerner.com/pull/732/commits/247bf907d4433dd23adfb5dd1438bd3ae0bd78a6#diff-8e226cf2e473b91b8c3db632af78280b2e61a4509894227d43e038cbd35eee68R241

File: lib/resources/r4/family_member_history.yaml (hunk 5 above)
Comment (sydneanschutz) [reply lib/resources/r4/family_member_history.yaml]: fixed all over https://github.com/cerner/fhir.cerner.com/commit/2b9dbafaf8d103502e3951c22745a5718eab499c#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L207

File: lib/resources/r4/family_member_history.yaml [hunk 6]
```diff
@@ -466,12 +480,31 @@ fields:
         }
       ]
     }
+  example2: |
+    {
+      "modifierExtensions": [
```
[the remark this answers is not in the document]
Comment (sydneanschutz) [reply lib/resources/r4/family_member_history.yaml]: https://github.com/cerner/fhir.cerner.com/pull/732/commits/fc3aa4bff12bae6b37db68fb4c2317ddc76483a8#diff-784415f7640df65b77af88df40ea89d5c05f009fbe1d256c1c8a0702baa87d63L469

**Review by MadhurRaj — APPROVED** [review]

**Review by RayGunY — APPROVED** [review]
looks good!

**Comment by NimishaMN** [comment]
Post deployment screenshots captured:
Screenshot 2021-09-23 at 9 38 32 PM
Screenshot 2021-09-23 at 9 38 53 PM
Screenshot 2021-09-23 at 9 39 57 PM
Screenshot 2021-09-23 at 9 40 36 PM
Screenshot 2021-09-23 at 9 41 07 PM

**Pull request merged**